committeeoversightapp/templates/partials/top_menu.html

```diff
@@ -0,0 +1,45 @@
+{% load static wagtailcore_tags cms_tags %}
+
+<nav class="navbar navbar-dark fixed-top footer navbar-expand-md">
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></i></span>
+  </button>
+  <a class="navbar-brand" href="/">
+    <img src="{% static 'images/lugarcenter.svg' %}" height="40" alt="The Lugar Center"/>
+  </a>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav ml-auto">
+      <li class="nav-item">
+        <a href="/">Home</a>
+      </li>
+      {% for menuitem in menuitems %}
+        <li class="nav-item {% if menuitem.show_dropdown %}dropdown{% endif %}{% if calling_page.url == menuitem.url %} active{% endif %}">
+            {% if menuitem.show_dropdown %}
+              <a data-toggle="dropdown" class="dropdown-toggle" href="#">{{ menuitem.title }} <b class="caret"></b></a>
+              {% top_menu_children parent=menuitem %}
+            {% else %}
+              <a href="{% pageurl menuitem %}">{{ menuitem.title }}</a>
+            {% endif %}
+        </li>
+      {% endfor %}
+      {% if request.user.is_superuser %}
```

Comment:
In the interest of not cluttering up the top nav, what do you think about putting "Add Hearing," "View Hearings," and "Logout" under a menu item called "Admin"?

Reply:
@beamalsky will there be non-admin user accounts?

Reply:
Nope! The only reason someone would be logging in right now is to create, edit, and delete hearings.